src/content/en/fundamentals/media/mse/basics.md

```diff
@@ -91,6 +91,11 @@ Here, in no particular order, are a few things I won't cover.
    `<audio>` and `<video>` elements.
 +  Error handling for the sake of simplicity and clarity.
 
+Warning: Handling error events, API exceptions, and proactively checking
+`HTMLMediaElement readyState` and `MediaSource readyState` values before making
+calls on their APIs are mandatory in a production usage of the APIs as these
+values can change before associated events are delivered.
+
```

Comment:
Thanks for this feedback @jpmedley @wolenetz.
WDYT of https://github.com/google/WebFundamentals/pull/4770/commits/6cd7a7aef69168da7b98a5f7b8f4c66b407361a9?